python_modules/dagster-graphql/dagster_graphql/schema/instigation.py

```diff
@@ -224,6 +224,14 @@ def resolve_skippedPartitionKeys(self, _graphene_info: ResolveInfo):
         return self._dynamic_partitions_request_result.skipped_partitions
 
 
+class GrapheneRequestedAssetMaterialization(graphene.ObjectType):
```

Comment:
I think I would expect this name to be plural, given that there can be multiple partition keys, and there's an AssetMaterialization event per partition key.

Reply:
this makes sense. is it confusing that requestedAssetMaterializations is then a list of GrapheneRequestedAssetMaterialization? 

requestedAssetMaterializationses?

Reply:
`GrapheneAssetMaterializationRequest` maybe? A "request" being implicitly possibly plural?

Reply:
I think I would still expect `GrapheneAssetMaterializationRequest` to refer to a single asset materialization.

Reply:
`GrapheneRequestedAssetMaterializations` / `allRequestedAssetMaterializations` mayhaps

Reply:
`GrapheneRequestedMaterializationsForAsset` and `requestedMaterializationsForAssets`?

`GrapheneAssetKeyAndPartitions` and `requestedAssetMaterializations`?

Reply:
Maybe something that brings in the fact that this is grouped by asset key. `GrapheneRequestedMaterializationsForAsset` / `requestedMaterializationsForAssets`

Reply:
@OwenKephart great minds think alike!

Reply:
haha congratulations on the earlier timestamp